application/pom.xml

```diff
@@ -264,6 +264,10 @@
             <groupId>org.javadelight</groupId>
             <artifactId>delight-nashorn-sandbox</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.springfox.ui</groupId>
+            <artifactId>springfox-swagger-ui-rfc6570</artifactId>
```

Comment:
No version is set. Not sure if this will compile. Version should be set in dependency management in main pom.xml

Reply:
Added version already

Reply:
This version should be added to main pom.xml into dependencyManagement and not to this child pom.xml (application). We keep all versions in one place!